app/src/models/publish-settings.ts

```diff
@@ -0,0 +1,37 @@
+import { IAPIUser } from '../lib/api'
+
+export type RepositoryPublicationSettings =
+  | IEnterprisePublicationSettings
+  | IDotcomPublicationSettings
+
+export interface IEnterprisePublicationSettings {
+  readonly kind: 'ghe'
```

Comment:
I'm still a fan of using `'enterprise'` here rather than `'ghe'` for these magic strings because I consider `GHE` an acronym but we've had to lower-case it here because it's basically an enum value. I also think `'enterprise'` reads better  alongside `'dotcom'`.

We also have some variables where I think `gheTabState` could be renamed to `enterpriseTabState` to stick to this pattern. Any objections?

Reply:
None here